Tracker/tracker.lua

```diff
@@ -9979,11 +9979,10 @@ local function updateLoop()
     end
   end
 
-  tracker:forceCursorInRange()
-  if ( tracker.cfg.followSong == 1 ) then
-    if ( reaper.GetPlayState() ~= 0 ) then
-      tracker:gotoCurrentPosition()
-    end
+  if ( tracker.cfg.followSong == 1 and reaper.GetPlayState() ~= 0) then
+    tracker:gotoCurrentPosition()

```

Comment:
Actually I've just realised that `gotoCurrentPosition` doesn't necessarily call `forceCursorInRange`, so a call to `forceCursorInRange` could get dropped with this change. I'm not sure how this might make an observable difference - but can drop this commit if it does.

Reply:
Btw, the reason I added this was I hacked in some `print()` lines in `forceCursorInRange` whilst working out what was wrong - I noticed it being called alternately with two different `yTarget` values which seemed odd.

Reply:
Should be ok.

The only time it wouldn't get called is if the item was lost, in which case clamping it wouldn't make much sense anyhow.

Reply:
Ah, just realized, forgot the version bump and changelog entries. I'll just copy the ones from your PR description :)

Thanks for the tweaks! :)

Reply:
Oh yeah, sorry, forgot about the version bump too. Cheers, thanks for merging!